### ISSUE TITLE
Paste images don't work.

### ISSUE THREAD

**Issue by DBachiniLima** [insin/control-panel-for-twitter#234]
I don't really know why it's happening, but I did the test in the incognito tab with the extension turned off: 

I can't 'paste' images with control v or by right-clicking and clicking 'paste'.

I use this a lot when I want to paste a quick print that I made, and although the extension is wonderful, I just can't do it with it active.

**Comment by insin** [comment]
Which browser and version the extension are you using? I can't think of anything Tweak New Twitter does which should affect this. Do you have any other Twitter extensions active as well?

Pasting both ways is working for me:

![firefox_xwPAGuDmsM](https://user-images.githubusercontent.com/226692/216777173-0f216280-9d8e-465c-86d3-10cd426c6670.gif)

**Comment by DBachiniLima** [comment]
Sorry for the delay in responding, I've had a pretty busy week. 
My browser was Chrome at 109.0.5414.120, and it just updated to the latest version.
I wanted to make sure the problem was just me. I discovered the extension conflict. I use one called "Absolute Enable Right Click & Copy" that allows you to right click on websites that don't allow it. When it is activated on twitter, combined with the Control Panel, a bug occurs, which does not allow me to do this. When I turn off one of the two, the function works again. I already got it working. Thank you very much for your answer, I hope that if someone has a similar problem, they can do what I did to disable each extension to discover the origin. I have a lot of extensions installed (27 at the moment), so it's always a problem to figure out what's going wrong.
